.github/workflows/automation-pull-upstream.yml

```diff
@@ -51,6 +51,20 @@ jobs:
           DEBIAN_FRONTEND=noninteractive sudo apt-get install -y \
             python3-requests ninja-build
 
+      # We had the following on gha:
+      # > You are running out of disk space. The runner will stop working when the machine runs out of disk space. Free space left: 0 MB
+      # Prune more if this happens again... there is plenty of unused things in the image.
+      - name: Free disk space
```

Comment:
Think we can throw this into a `ferrocene/ci/scripts/gha-disk-free-up.sh`? I had to do similar in https://github.com/ferrocene/ferrocene/blob/b59fe98749df00463b5269004aa5255b9e20dd2c/ferrocene/ci/scripts/setup-darwin.sh#L24 on the GHA migration and it'd be nice to have a unified place to do this, as I expect other jobs will want it.